app.py

```diff
@@ -40,12 +40,12 @@ def confirm_use(token):#确定一下Key有没有被用过
 def token():
     message = request.args.get('token')
     person_info = confirm_token(message)
-    if person_info:
-        return_json = {'code': 0, 'data': person_info,
-                       'message': 'success'}
+    if person_info and person_info['status'] == 1:
```

Comment:
status == 1 is success, contrary to the original. This should be reflected in documentation also.

Reply:
status == 1  means  ：the  mail is sent successfully，  when the submitUserInfo  get data from user interface ,  it should be this status

Reply:
No, your understand is probably not correct. Currently, status == 1 means the email has been sent before.